src/openrct2/actions/FootpathPlaceAction.hpp

```diff
@@ -185,12 +185,19 @@ DEFINE_GAME_ACTION(FootpathPlaceAction, GAME_COMMAND_PLACE_PATH, GameActionResul
         if (_type & FOOTPATH_ELEMENT_INSERT_QUEUE)
         {
             pathElement->SetIsQueue(true);
+
+            //remove all but TVs
+            if (pathElement->GetAddition() != 0x6 && pathElement->GetAddition() != 0x5)
```

Comment:
Please use constants rather than hardcoded numbers.

Reply:
I am not sure how to define those constants to be honest. Sometimes the TV addition put a value of 0x5 sometimes 0x6... Should I declare constants ADDITION_TV_1 = 0x5 and ADDITION_TV_2 = 0x6?

Reply:
@frutiemax the reason they change is because they are not constant. These are index's into the objects. What you should do is use GetAdditionEntryIndex and use that to index the object. Then the object will tell you what type it is.